ksql-cli/src/main/java/io/confluent/ksql/cli/Cli.java

```diff
@@ -238,20 +238,23 @@ public void close() throws IOException {
     terminal.close();
   }
 
-  public void handleLine(String line) throws Exception {
-    String trimmedLine = Optional.ofNullable(line).orElse("").trim();
+  public void handleLine(final String line) throws Exception {
+    // trim whitespace and convert non-breaking spaces to spaces;
+    final String trimmedLine = Optional.ofNullable(line).orElse("")
+        .replace((char) 0xa0, ' ')
```

Comment:
This only handlings non-breaking spaces; what about tabs?

How about using `.replaceAll("\\s+", " ")`, (I'm assuming `\\s+` will include non-breaking spaces, as well as tabs etc.

Reply:
This may unintentionally change the string literal values surrounded by single quote! Is there any other way to exclude them?

Reply:
Good point. Can Antlr not handle other whitespace types other than a simple space?